src/main/java/stringcalculator/domain/StringUtils.java

```diff
@@ -0,0 +1,47 @@
+package stringcalculator.domain;
+
+public class StringUtils {
+
```

Comment:
```suggestion
public class StringUtils {

    private StringUtils() {}
```

유틸 클래스는 객체 생성이 불가능하도록 생성자를 private 로 해주세요